python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputSchemaRenderer.tsx

```diff
@@ -61,15 +62,17 @@ function SchemaRenderer({ input, schema, onChangeInput }: SchemaRendererProps) {
 
 export default memo(function PromptInputSchemaRenderer(props: Props) {
   if (props.schema.type === "string") {
-    // TODO: Add ErrorBoundary handling
-    // if (props.input && typeof props.input !== "string") {
-    //   throw new Error(
-    //     `Expected input to be a string, but got ${typeof props.input}`
-    //   );
-    // }
+    if (props.input && typeof props.input !== "string") {
```

Comment:
For future, this means we can't have it be of general type `PromptInput` I guess? But I supposed for now that's fine as long as we support attachments directly later

Reply:
For this case, if there is a Prompt Schema defined for the input and that schema specifies the type should be string then the actual input type should match 'string' instead of being an object